src/aerospike_clj/client.clj

```diff
@@ -63,27 +64,31 @@
   \"username\"
   :port (default is 3000)
   :client-events an implementation of ClientEvents. Either a single one or a vector
-  thereof. In the case of a vector, the client will chain the instances by order."
+  thereof. In the case of a vector, the client will chain the instances by order.
+  If no event-loops instance is provided, single-threaded one is created automatically."
```

Comment:
provided,  **a** single-threaded

Reply:
Updated the docs, please review and if everything is fine, please resolve.